src/App.css

```diff
@@ -313,24 +313,27 @@ a {
 #drag-group,
 .piece {
   display: grid;
-  grid-template-columns: repeat(var(--grid-columns), var(--box-size));
-  grid-template-rows: repeat(var(--grid-rows), var(--box-size));
-  width: calc(var(--grid-columns) * var(--box-size));
-  height: calc(var(--grid-rows) * var(--box-size));
+  grid-template-columns: repeat(var(--grid-columns), var(--box-size)) 1px;
+  grid-template-rows: repeat(var(--grid-rows), var(--box-size)) 1px;
```

Comment:
I added this extra 1px row and 1px column to cause the grid to be the exact same size as the `width` and `height` of the container.

Without this, the browser centers the grid within the container, so there is a 0.5px margin at the right and top; it looks terrible.